### PR TITLE
Added some utility classes for interacting with the database.

### PR DESCRIPTION
**Pull request by sammyers**

### CODE REVIEW

File: app/src/main/java/pizza/olin/consamables/database/DataHandler.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ */
+
+public class DataHandler {
+    private DatabaseReference mDatabase;
+
+    public DataHandler() {
+        mDatabase = FirebaseDatabase.getInstance().getReference();
+    }
+
+    public void createOrder(int duration) {
```
Comment (wolfd) [inline app/src/main/java/pizza/olin/consamables/database/DataHandler.java:21]: Units?

File: app/src/main/java/pizza/olin/consamables/types/Order.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
+package pizza.olin.consamables.types;
+
+import java.util.Date;
+
+/**
+ * Created by Sam on 10/11/2016.
+ */
```
Comment (wolfd) [inline app/src/main/java/pizza/olin/consamables/types/Order.java]: Tell Android Studio to not add these comments

File: app/src/main/java/pizza/olin/consamables/types/User.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
+package pizza.olin.consamables.types;
+
+/**
+ * Created by Sam on 10/11/2016.
+ */
+
+public class User {
```
Comment (wolfd) [inline app/src/main/java/pizza/olin/consamables/types/User.java]: We may not want our own user class, especially if we have FirebaseUser

**Review by wolfd — CHANGES_REQUESTED** [review]
The units for the time are not clear, I would change `duration` to `durationMinutes` if you want minutes.

File: app/src/main/java/pizza/olin/consamables/database/DataHandler.java (hunk 1 above)
Comment (sammyers) [reply app/src/main/java/pizza/olin/consamables/database/DataHandler.java:21]: Fixed in latest commit.

File: app/src/main/java/pizza/olin/consamables/types/Order.java (hunk 2 above)
Comment (sammyers) [reply app/src/main/java/pizza/olin/consamables/types/Order.java]: K.

File: app/src/main/java/pizza/olin/consamables/types/User.java (hunk 3 above)
Comment (sammyers) [reply app/src/main/java/pizza/olin/consamables/types/User.java]: Agreed.

**Review by wolfd — APPROVED** [review]

**Pull request merged**